src/ResourceManager/Kubernetes/Commands.Kubernetes/Commands/StartAzureRmKubernetesDashboard.cs

```diff
@@ -0,0 +1,346 @@
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Management.Automation;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+using Microsoft.Azure.Commands.Kubernetes.Generated;
+using Microsoft.Azure.Commands.Kubernetes.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+#if NETSTANDARD
+using Microsoft.Extensions.DependencyInjection;
+
+#endif
+
+namespace Microsoft.Azure.Commands.Kubernetes
+{
+    [Cmdlet("Start", KubeNounStr + "Dashboard")]
+    [OutputType(typeof(KubeTunnelJob))]
+    public class StartAzureRmKubernetesDashboard : KubeCmdletBase
+    {
+
+        private const string IdParameterSet = "IdParameterSet";
+        private const string GroupNameParameterSet = "GroupNameParameterSet";
+        private const string InputObjectParameterSet = "InputObjectParameterSet";
+
+        [Parameter(Mandatory = true,
+            ParameterSetName = InputObjectParameterSet,
+            ValueFromPipeline = true,
+            HelpMessage = "A PSKubernetesCluster object, normally passed through the pipeline.")]
+        [ValidateNotNullOrEmpty]
+        public PSKubernetesCluster InputObject { get; set; }
+
+        /// <summary>
+        /// Cluster name
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = IdParameterSet,
+            Position = 0,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Id of a managed Kubernetes cluster")]
+        [ValidateNotNullOrEmpty]
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Cluster name
+        /// </summary>
+        [Parameter(
+            Mandatory = true,
+            Position = 0,
+            ParameterSetName = GroupNameParameterSet,
+            HelpMessage = "Name of your managed Kubernetes cluster")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Resource group name
+        /// </summary>
+        [Parameter(
+            Position = 1,
+            Mandatory = true,
+            ParameterSetName = GroupNameParameterSet,
+            HelpMessage = "Resource group name")]
+        [ResourceGroupCompleter()]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = false,
+            HelpMessage = "Do not pop open a browser after establising the kubectl port-forward.")]
+        public SwitchParameter DisableBrowser { get; set; }
+
+        [Parameter(Mandatory = false)]
+        public SwitchParameter PassThru { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            base.ExecuteCmdlet();
+
+            switch (ParameterSetName)
+            {
+                case IdParameterSet:
+                    {
```

Comment:
no reason for braces here

Reply:
It is needed due to 'var resource = ...' in both blocks. This limits scope.